streampipes-data-explorer-commons/src/main/java/org/apache/streampipes/dataexplorer/commons/influx/InfluxStore.java

```diff
@@ -218,6 +226,21 @@ private void handleMeasurementProperty(Point.Builder p,
     }
   }
 
+  private void handleNonPrimitiveMeasurementProperty(Point.Builder p, Event event, String preparedRuntimeName) {
+    var field = event.getRaw().get(preparedRuntimeName);
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
I think we can change the object mapper as a class variable.

Reply:
Will do.